.dotnet/src/Polyfill/System.Diagnostics.CodeAnalysis.ExperimentalAttribute.cs

```diff
@@ -0,0 +1,59 @@
+#if !NET8_0_OR_GREATER
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>
+    ///  Indicates that an API is experimental and it may change in the future.
+    /// </summary>
+    /// <remarks>
+    ///   This attribute allows call sites to be flagged with a diagnostic that indicates that an experimental
+    ///   feature is used. Authors can use this attribute to ship preview features in their assemblies.
+    /// </remarks>
+    [AttributeUsage(AttributeTargets.Assembly |
+                    AttributeTargets.Module |
+                    AttributeTargets.Class |
+                    AttributeTargets.Struct |
+                    AttributeTargets.Enum |
+                    AttributeTargets.Constructor |
+                    AttributeTargets.Method |
+                    AttributeTargets.Property |
+                    AttributeTargets.Field |
+                    AttributeTargets.Event |
+                    AttributeTargets.Interface |
+                    AttributeTargets.Delegate, Inherited = false)]
+    public sealed class ExperimentalAttribute : Attribute
```

Comment:
does it need to be public?

Reply:
In fact it should not be public. Polyfills like this should always be internal to avoid conflicts when multiple assemblies in a compilation bring their own polyfills.

Reply:
Absolutely -- thanks for catching this. I had it internal [in the original commit](https://github.com/joseharriaga/openai-in-typespec/pull/17/commits/27abab53712975b9695755571d3293d4415b5f83) but missed restoring the visibility modifier after updating to the better source version that Jose recommended.